{"lang":"ja","messages":[{"text_key":"reservation-incorrect-step-key","source":"Task is no longer current, probably already completed. Please refresh your Inbox.","target":"タスクは最新ではなく、おそらくすでに完了しています。受信トレイをリフレッシュしてください。"},{"text_key":"reservation-lock-timeout","source":"Subflow currently locked (not reserved) by another user. Try your reservation again later.","target":"サブフローは別のユーザーによって現在ロックされています(予約されていません)。後で予約を再試行してください。"},{"text_key":"reservation-release-not-found","source":"Reservation release unsuccessful. Subflow %0 in Process %1 not found.","target":"予約のリリースに失敗しました。プロセス%1のサブフロー%0が見つかりません。"},{"text_key":"restart-no-error","source":"No Current Error Found. Check your process diagram.","target":"現在のエラーが見つかりません。プロセス・ダイアグラムを確認します。"},{"text_key":"start-already-running","source":"You tried to start a process (id %0)that is already running.","target":"プロセス (id %0) は既に実行中です。"},{"text_key":"start-diagram-calls-itself","source":"You tried to start a process with a diagram %0 that includes a callActivity calling itself.","target":"コールアクティビティを含むダイアグラム%0でプロセスを開始しようとしました。"},{"text_key":"start-multiple-already-running","source":"You tried to start a process (id %0) with multiple copies already running.","target":"プロセス (id %0) において既に複数のコピーが実行中です。"},{"text_key":"start-multiple-start-events","source":"You have multiple starting events defined. Make sure your diagram has only one start event.","target":"複数の開始イベントが定義されています。ダイアグラムの開始イベントが1つのみとしてください。"},{"text_key":"start-no-start-event","source":"No starting event is defined in the Flow diagram.","target":"フロー図に開始イベントが定義されていません。"},{"text_key":"start-not-created","source":"You tried to start a process (id %0) that does not exist.","target":"プロセス (id %0) は存在しないため開始できません。"},{"text_key":"async-no-username","source":"Unable to create asyncronous connection for object %0. Username needs to be specified in process variable, the process diagram, or system configuration.","target":"オブジェクト%0の非同期接続を作成できません。プロセス変数、プロセス図、またはシステム構成にユーザー名を指定する必要があります。"},{"text_key":"async-no-appid","source":"Unable to create asyncronous connection for object %0. Application ID needs to be specified in process variable, the process diagram, or system configuration.","target":"オブジェクト%0の非同期接続を作成できません。アプリケーションIDは、プロセス変数、プロセス図またはシステム構成で指定する必要があります。"},{"text_key":"step-key-incorrect","source":"This Process Step has already occurred. (Incorrect step key %0 supplied while expecting step key %1).","target":"このプロセス・ステップはすでに実行されています。(ステップ・キー%1を予期している間に間違ったステップ・キー%0が指定されました)。"},{"text_key":"subProcess-no-start","source":"Unable to find Sub-Process Start Event.","target":"サブプロセス開始イベントが見つかりません。"},{"text_key":"settings-procvar-no-prcs","source":"Settings cannot specify Process Variable without a Process ID.","target":"設定では、プロセスIDなしでプロセス変数を指定できません。"},{"text_key":"diagram-archive-has-instances","source":"You tried to archive a diagram that has running instances.","target":"実行中のインスタンスがあるダイアグラムをアーカイブしようとしました。"},{"text_key":"logging-diagram-event","source":"Flows - Internal error while logging a Diagram Event.","target":"フロー- ダイアグラム・イベントのロギング中に内部エラーが発生しました。"},{"text_key":"log-archive-error","source":"Error archiving instance summary for Process Instance %0","target":"プロセス・インスタンス%0のインスタンス要約のアーカイブ中にエラーが発生しました"},{"text_key":"msgflow-lock-timeout-subflow","source":"Message receiver unable to lock subflow. Try again.","target":"メッセージ受信者はサブフローをロックできません。再試行してください。"},{"text_key":"settings-error","source":"Error evaluating Setting. Expression is invalid. Expression: %0.","target":"設定の評価中にエラーが発生しました。式が無効です。式: %0。"},{"text_key":"msgflow-endpoint-not-supported","source":"MessageFlow Endpoint specified ( %0 ) is not supported.","target":"MessageFlow指定されたエンドポイント( %0 )はサポートされていません。"},{"text_key":"msgflow-lock-timeout-msub","source":"Message Subscription locked by another user. Try again.","target":"メッセージ・サブスクリプションは別のユーザーによってロックされています。再試行してください。"},{"text_key":"apex-task-business-ref-null","source":"Error creating Approval Task - Business Ref / System of Record Primary Key must be not null.","target":"承認タスクの作成中にエラーが発生しました- ビジネス参照/レコードのシステム主キーはnullにできません。"},{"text_key":"due-on-interval-error","source":"Error evaluating Due On. Interval expression is invalid. Interval: %0.","target":"期日の評価中にエラーが発生しました。間隔式が無効です。間隔: %0。"},{"text_key":"due-on-error","source":"Error evaluating Due On. Due On expression is invalid. Interval Expression: %0. ","target":"期日評価の際にエラーが出ました。期日表示が無効です。間隔表示：%0"},{"text_key":"settings-priority-error","source":"Error evaluating Priority. Priority expression is invalid. Expression: %0. Priority 3 used instead.","target":"優先順位の評価中にエラーが発生しました。優先度式が無効です。式: %0。かわりに優先度3が使用されます。"},{"text_key":"var-bad-scope","source":"Invalid scope (%0) supplied for process variable.","target":"プロセス変数に無効なスコープ(%0)が指定されました。"},{"text_key":"scheduler-repeat-shared-env","source":"Timer repeat interval too frequent for host (%0) Requested Interval %1. Must be greater than 1 Minute.","target":"ホスト(%0)が要求した間隔%1に対してタイマー繰返し間隔が頻繁すぎます。1分より大きくする必要があります。"},{"text_key":"archive-destination-bad-json","source":"Error in archive destination configuration parameter. Parameter: %0","target":"アーカイブ先構成パラメータでエラーが発生しました。パラメータ: %0"},{"text_key":"msgflow-not-correlated","source":"Received message does not match an expected message.","target":"受信したメッセージが予期されたメッセージと一致しません。"},{"text_key":"msgflow-no-longer-current-step","source":"Message correlated but receiving object is no longer the current object in its subflow.","target":"メッセージは相関していますが、受信オブジェクトはサブフロー内の現在のオブジェクトではなくなりました。"},{"text_key":"start-type-unsupported","source":"Unsupported start event type (%0). Only None (standard), Message and Timer Start Event are currently supported.","target":"スタートイベントタイプがサポートされていません。(%0 ) 現在、標準メッセージおよびタイマースタートイベントのみサポートされています。"},{"text_key":"startwork-sbfl-not-found","source":"Start Work time recording unsuccessful. Subflow %0 in Process %1 not found.","target":"勤務時間の記録の開始に失敗しました。プロセス%1のサブフロー%0が見つかりません。"},{"text_key":"feature-requires-ee","source":"Processing this feature requires licensing Flows for APEX Enterprise Edition.","target":"この機能を処理するには、Flow for APEX Enterprise Editionのライセンスが必要です。"},{"text_key":"iter_close_error","source":"Internal error processing iteration closure on subflow %0","target":"サブフロー%0の反復クローズの処理中に内部エラーが発生しました"},{"text_key":"var_exp_json_format","source":"Error setting Process Variable %1: Incorrect JSON Format (Subflow: %0, Set: %3.)","target":"プロセス変数%1の設定中にエラーが発生しました: JSONフォーマットが正しくありません(サブフロー: %0、セット: %3)。"},{"text_key":"iter_lock_error","source":"Error locking iteration detail (iter %0 loop %2) for process id %1.","target":"プロセスID %1の反復詳細(反復%0ループ%2)をロック中にエラーが発生しました。"},{"text_key":"apex-task-invalid-result-var","source":"APEX Human Task result process variable is not defined or invalid.","target":"APEXヒューマン・タスク結果プロセス変数が定義されていないか、無効です。"},{"text_key":"async-no-pageid","source":"Unable to create asyncronous connection for object %0. Page ID needs to be specified in process variable, the process diagram, or system configuration.","target":"オブジェクト%0の非同期接続を作成できません。ページIDは、プロセス変数、プロセス図またはシステム構成で指定する必要があります。"},{"text_key":"boundary-event-child-lock-to","source":"Child Boundary Subflows or Timers of %0 currently locked by another user. Retry your transaction later.","target":"%0の子境界サブフローまたはタイマーは、現在別のユーザーによってロックされています。後でトランザクションを再試行してください。"},{"text_key":"logging-variable-event","source":"Flows - Internal error while logging a Variable Event","target":"フロー - 変数イベントのログ記録時に内部エラーが発生しました。"},{"text_key":"process-lock-timeout","source":"Process objects for %0 currently locked by another user. Try again later.","target":"%0の処理オブジェクトは、現在別のユーザーによってロックされています。後で再試行してください。"},{"text_key":"more_than_1_forward_path","source":"More than 1 forward path found when only 1 allowed.","target":"複数のフォワードパスが見つかりました。パスは一つのみ可能です。"},{"text_key":"no_next_step_found","source":"No Next Step Found on subflow %0. Check your process diagram.","target":"サブフロー%0に次のステップが見つかりません。プロセス・ダイアグラムを確認します。"},{"text_key":"plsql_script_failed","source":"Process %0: Task %1 failed due to PL/SQL error - see event log.","target":"プロセス %0: PL/SQL エラーが発生したため、タスク％１に失敗しました。イベント・ログを参照してください。"},{"text_key":"plsql_script_requested_stop","source":"Process %0: Task %1 requested processing stop - see event log.","target":"プロセス %0: タスク％１が処理の停止を要求しました - イベント・ログを参照してください。"},{"text_key":"timeout_locking_subflow","source":"Unable to lock subflow %0 as currently locked by another user. Try again later.","target":"サブフロー%0は、現在別のユーザーによってロックされているためロックできません。後で再試行してください。"},{"text_key":"reservation-already-placed","source":"Reservation already placed on next task for you.","target":"次のタスクの予約はすでにされています。"},{"text_key":"reservation-by-other_user","source":"Reservation for %0 unsuccessful. Step already reserved by another user (%1).","target":"%0の予約に失敗しました。ステップは別のユーザー(%1)によってすでに予約されています。"},{"text_key":"reservation-failed-not-found","source":"Reservation for %2 unsuccessful. Subflow %0 in Process %1 not found.","target":"%2の予約に失敗しました。プロセス%1のサブフロー%0が見つかりません。"},{"text_key":"subProcess-too-many-starts","source":"More than one Sub-Process Start found.","target":"複数のサブプロセス・スタートが見つかりました。"},{"text_key":"timer-broken","source":"Timer %0 Run %4 broken in process %1 , subflow : %2. See error_info.","target":"タイマー%0実行%4がプロセス%1、サブフロー: %2で破損しています。error_infoを参照してください。"},{"text_key":"timer-incomplete-definition","source":"Incomplete timer definitions for object %0. Type: %1; Value1: %2 Value2: %3 Value3: %4","target":"オブジェクト%0のタイマー定義が不完全です。タイプ: %1; Value1: %2 Value2: %3 Value3: %4"},{"text_key":"timer-lock-timeout","source":"Timer for subflow %0 currently locked by another user. Try again later.","target":"サブフロー%0のタイマーは現在別のユーザーによってロックされています。後で再試行してください。"},{"text_key":"timer-object-not-found","source":"Object with timer not found in get_timer_definition. Subflow %0.","target":"オブジェクトのタイマーが get_timer_definition で見つかりませんでした。サブフロー %0。"},{"text_key":"timers-lock-timeout","source":"Timers for process %0 currently locked by another user. Try again later.","target":"プロセス%0のタイマーは現在別のユーザーによってロックされています。後で再試行してください。"},{"text_key":"var-set-error","source":"Error creating process variable %0 for process id %1 in scope %2.","target":"スコープ%2のプロセスID %1のプロセス変数%0を作成中にエラーが発生しました。"},{"text_key":"var-get-error","source":"Error getting process variable %0 for process id %1 in scope %2.","target":"スコープ%2のプロセスID %1のプロセス変数%0を取得中にエラーが発生しました。"},{"text_key":"var-update-error","source":"Error updating process variable %0 for process id %1 in scope %2.","target":"スコープ%2のプロセスID %1のプロセス変数%0を更新中にエラーが発生しました。"},{"text_key":"var-delete-error","source":"Error deleting process variable %0 for process id %1 in scope %2.","target":"スコープ%2のプロセスID %1のプロセス変数%0を削除中にエラーが発生しました。"},{"text_key":"var-lock-error","source":"Error locking process variable %0 for process id %1 in scope %2.","target":"スコープ%2のプロセスID %1のプロセス変数%0のロック中にエラーが発生しました。"},{"text_key":"var_exp_datatype","source":"Error setting process variable. Incorrect datatype for variable %0. SQL error shown in debug output.","target":"プロセス変数の設定中にエラーが発生しました。変数%0のデータ型が正しくありません。デバッグ出力にSQLエラーが表示されます。"},{"text_key":"var_exp_date_format","source":"Error setting Process Variable %1: Incorrect Date Format (Subflow: %0, Set: %3.)","target":"プロセス変数 %1 の設定エラー : 日付の形式が正しくありません (サブフロー: %0、設定: %3)"},{"text_key":"var_exp_object_not_found","source":"Internal error looking up object %0 in process_expressions. SQL error shown in debug output.","target":"process_expressionsでオブジェクト%0を検索中に内部エラーが発生しました。デバッグ出力にSQLエラーが表示されます。"},{"text_key":"var_exp_plsql_error","source":"Subflow : %0 Error in %2 expression for Variable : %1","target":"サブフロー :% 0 変数 : %1の%2においてエラーが発生しました。"},{"text_key":"var_exp_sql_no_data","source":"Error setting process variable %1 in process id %0 (set %2). No data found in query.","target":"プロセスID %0 (セット%2)でプロセス変数%1を設定中にエラーが発生しました。問合せにデータが見つかりません。"},{"text_key":"var_exp_sql_other","source":"Error setting process variable %1 in process id %0 (set %2). SQL error shown in event log.","target":"プロセスID %0 (セット%2)でプロセス変数%1を設定中にエラーが発生しました。イベント・ログにSQLエラーが表示されます。"},{"text_key":"var_exp_static_general","source":"Error setting process variable %1 in process id %0 (set %2). See error in event log.","target":"プロセスID %0 (セット%2)でプロセス変数%1を設定中にエラーが発生しました。イベントログのエラーを参照してください。"},{"text_key":"version-no-rel-or-draft-v0","source":"Cannot find released diagram or draft version 0 of diagram - please specify a version or diagram_id","target":"リリースされたダイアグラムまたはドラフト版0が見つかりません - バージョンまたはdiagram_idを指定してください。"},{"text_key":"version-not-found","source":"Cannot find specified diagram version. Please check version specification.","target":"指定されたダイアグラム・バージョンが見つかりません。バージョン指定を確認してください。"},{"text_key":"timer_definition_error","source":"Error parsing timer definition in process %0, subflow %1. Timer Type: %2, Definition: %3","target":"プロセス %0、サブフロー %1 でタイマー定義の解析にエラーが発生しました。タイマータイプ: %2, 定義: %3"},{"text_key":"plugin-model-no-version","source":"Version not defined.","target":"バージョンが定義されていません。"},{"text_key":"plugin-parsing-json-variables","source":"Error during parsing process variables.","target":"プロセス変数のパーシング中にエラーが発生しました。"},{"text_key":"plugin-route-not-define","source":"Gateway is not defined for routing.","target":"ゲートウェイがルーティング用に定義されていません。"},{"text_key":"plugin-gateway-not-exist","source":"Gateway define does not exists for this flow.","target":"このフローにはゲートウェイ定義が存在しません。"},{"text_key":"timer-cycle-unsupported","source":"Cycle Timer defined for object %0 not currently supported.","target":"オブジェクト %0 で定義されたサイクル・タイマーは、現在サポートされていません。"},{"text_key":"plugin-no-instance-subflow-id","source":"Unable to get Flow Instance Id and or subflow Id to manage the step.","target":"ステップを管理するためのフローインスタンスID、またはサブフローIDを取得できません。"},{"text_key":"plugin-wrong-variable-number","source":"Wrong number of APEX item(s) or process variable(s).","target":"APEX項目またはプロセス変数の数が間違っています。"},{"text_key":"plugin-wrong-variable-type","source":"One or more process variable(s) are a different type than the one defined in the JSON.","target":"1つ以上のプロセス変数が、JSONで定義されたものと異なるタイプです。"},{"text_key":"plugin-variable-not-a-number","source":"%0 is not a valid number.","target":"0 は有効な数値ではありません。"},{"text_key":"plugin-variable-not-a-date","source":"%0 is not a valid date.","target":"0 は有効な日付ではありません。"},{"text_key":"plugin-invalid-json","source":"JSON provided is invalid.","target":"提供されたJSONは無効です。"},{"text_key":"plugin-modeler-id-not-found","source":"No data found. Check if Diagram with given ID exists.","target":"データが見つかりませんでした。指定されたIDのダイアグラムが存在するかどうかを確認してください。"},{"text_key":"plugin-unexpected-error","source":"Unexpected error, please contact your administrator.","target":"例外エラー発生。管理者に連絡してください。"},{"text_key":"plugin-diagram-not-parsable","source":"Diagram could not be parsed.<br />Please review your diagram to ensure that it is supported.","target":"ダイアグラムをパースすることができませんでした。このダイアグラムがサポートされているか確認してください。"},{"text_key":"plugin-diagram-saved","source":"Changes saved!","target":"変更が保存されました。"},{"text_key":"plugin-diagram-has-changed","source":"Model has changed. Discard changes?","target":"モデルが変更になりました。変更してよろしいですか。"},{"text_key":"wrong-default-workspace","source":"Process %0: ServiceTask %1 failed: the default workspace defined in the configuration parameter is not valid.","target":"プロセス %0: ServiceTask %1 に失敗しました: 構成パラメータで定義された既定のワークスペースが無効です。"},{"text_key":"workspace-not-found","source":"Process %0: ServiceTask %1 failed: unable to find the workspace associated with the application id defined in the diagram. Please check the model.","target":"プロセス %0: ServiceTask %1 に失敗しました: ダイアグラムで定義されているアプリケーション ID に関連付けられたワークスペースが見つかりませ ん。モデルを確認してください。"},{"text_key":"email-no-from","source":"Process %0: ServiceTask %1 failed: attribute \"From\" and default email sender are not defined. Please check the model and or the configuration parameter.","target":"プロセス %0: ServiceTask %1 に失敗しました。: \"From\" と既定のEメール送信者が設定されていません。 モデルまたは設定パラメータを確認してください。"},{"text_key":"email-no-to","source":"Process %0: ServiceTask %1 failed: attribute \"To\" not defined. Please check the model.","target":"プロセス %0: サービスタスク %1 に失敗しました。: \"To\" が設定されていません。モデルを確認してください。"},{"text_key":"email-no-template","source":"Process %0: ServiceTask %1 failed: attribute \"Template\" not defined. Please check the model.","target":"プロセス %0: サービスタスク%1に失敗しました。: \"Template\" が設定されていません。モデルを確認してください。"},{"text_key":"email-no-body","source":"Process %0: ServiceTask %1 failed: attribute \"Body\" not defined. Please check the model.","target":"プロセス %0: サービスタスク%1に失敗しました。: \"Body\" が設定されていません。モデルを確認してください。"},{"text_key":"email-failed","source":"Process %0: ServiceTask %1 failed see error log and check the model.","target":"プロセス %0: サービスタスク%1に失敗しました。エラー・ログを参照して、モデルを確認してください。"},{"text_key":"email-placeholder-json-invalid","source":"Process %0: ServiceTask %1 placeholder JSON is invalid.Please check the model.","target":"プロセス %0: サービスタスク プレースホルダー JSON が無効です。モデルを確認してください。"},{"text_key":"plugin-multiple-rows","source":"Multiple rows found. Please enable the 'Enable Call Activities' setting in the viewer plugin attributes.","target":"複数の行が見つかりました。ビューア・プラグイン属性で「コール・アクティビティの有効化」に設定してください。"},{"text_key":"apex-task-cancelation-error","source":"Error attempting to cancel APEX Human Task (task_id: %1 ) for process step : %0.)","target":"プロセス・ステップ: %0のAPEXヒューマン・タスク(task_id: %1)を取り消そうとしてエラーが発生しました。"},{"text_key":"apex-task-not-found","source":"APEX Human Task %0 not found in Flows for APEX Process","target":"APEXヒューマン・タスク%0がFlows for APEXプロセスに見つかりません"},{"text_key":"gateway-merge-error","source":"Internal error processing merging gateway on subflow %0","target":"サブフロー %0 でマージ・ゲートウェイを処理する際の内部エラー。"},{"text_key":"gateway-no-route","source":"No gateway routing instruction provided in variable %0 and model contains no default route.","target":"変数 %0 でゲートウェイ・ルーティングが指定されていなく、モデルにはデフォルト・ルートがありません。"},{"text_key":"gateway-bad-expression","source":"Bad gateway routing expression. This can occur if you attempt to bind a variable with embedded colon (:).","target":"不正なゲートウェイ・ルーティングです。これは、埋込みコロン(: )で変数をバインドしようとした場合に発生する可能性があります。"},{"text_key":"gateway-too-many-defaults","source":"More than one default route specified in model for Gateway %0.","target":"ゲートウェイ %0 のモデルで複数の既定ルートが指定されています。"},{"text_key":"link-no-catch","source":"Unable to find matching link catch event named %0.","target":"一致するリンク・キャッチ・イベント名 %0 を見つけることができません。"},{"text_key":"link-too-many-catches","source":"More than one matching link catch event named %0.","target":"一致するリンク・キャッチ・イベント名が %0 のものが複数あります。"},{"text_key":"logging-instance-event","source":"Flows - Internal error while logging an Instance Event","target":"Flows - インスタンスイベントのログ記録時に内部エラーが発生しました。"},{"text_key":"logging-step-event","source":"Flows - Internal error while logging a Step Event","target":"フロー - ステップイベントのログ記録時に内部エラーが発生しました。"},{"text_key":"ITE-unsupported-type","source":"Currently unsupported type of Intermediate Throw Event encountered at %0 .","target":"現在サポートされていないタイプの中間スローイベントが %0 で発生しました。"},{"text_key":"apex-session-params-not-set","source":"Asynchronous connection details for object %0 need to be set in process variables, diagram async connection details, or system config details.","target":"オブジェクト%0の非同期接続詳細は、プロセス変数、ダイアグラム非同期接続詳細またはシステム構成詳細で設定する必要があります。"},{"text_key":"async-invalid_params","source":"Unable to create asyncronous connection for object %0. Username not valid in specified Application","target":"オブジェクト%0の非同期接続を作成できません。指定されたアプリケーションでユーザー名が無効です"},{"text_key":"apex-task-on-multiple-steps","source":"APEX Human Task %0 found associated with more than one Flows for APEX process step.","target":"複数のFlows for APEXプロセス・ステップに関連付けられたAPEXヒューマン・タスク%0が見つかりました。"},{"text_key":"apex-task-not-current-step","source":"APEX Human Task %0 is not the current step of the Process. Step may have completed all ready.","target":"APEXヒューマン・タスク%0はプロセスの現在のステップではありません。ステップがすべて完了した可能性があります。"},{"text_key":"apex-task-creation-error","source":"Error creating APEX Human task %0 in application %1. see debug for details.","target":"アプリケーション%1でAPEXヒューマン・タスク%0を作成中にエラーが発生しました。詳細はデバッグを参照してください。"},{"text_key":"apex-task-priority-error","source":"Error evaluating Priority. Priority must be between 1 and 5. Priority: %0.","target":"優先順位の評価中にエラーが発生しました。優先度は1から5の間にする必要があります。優先度: %0。"},{"text_key":"apex-task-multiple-processes","source":"Error creating APEX session. BPMN diagram contains multiple Process objects.","target":"APEXセッションの作成中にエラーが発生しました。BPMNダイアグラムに、複数のプロセス・オブジェクトが含まれています。"},{"text_key":"call-diagram-not-callable","source":"You tried to call a diagram %0 that is marked as being not callable.","target":"コール不可のダイアグラム%0をコールしようとしました。"},{"text_key":"boundary-event-no-catch-found","source":"No boundaryEvent of type %0 found to catch event.","target":"イベントを捕捉するためのタイプ %0 の boundaryEvent が見つかりませんでした。"},{"text_key":"boundary-event-too-many","source":"More than one %0 boundaryEvent found on sub process.","target":"サブプロセスで 1 つ以上の %0 boundaryEvent が見つかりました。"},{"text_key":"eng_handle_event_int","source":"Flow Engine Internal Error: Process %0 Subflow %1 Module %2 Current %4 Current Tag %3","target":"フロー・エンジン内部エラー：プロセス %0 サブフロー%1 モジュール%2 現行 %4現行タグ %3"},{"text_key":"engine-unsupported-object","source":"Model Error: Process BPMN model next step uses unsupported object %0","target":"モデル・エラー。プロセス BPMN モデルの次のステップでサポートされていないオブジェクト %0 が使用されています。"},{"text_key":"engine-util-prcs-not-found","source":"Application Error: Process ID %0 not found.","target":"アプリケーション・エラー。プロセス ID %0 が見つかりませんでした。"},{"text_key":"engine-util-sbfl-not-in-prcs","source":"Application Error: Subflow ID supplied ( %0 ) exists but is not child of Process ID Supplied ( %1 ).","target":"アプリケーション・エラー。指定されたサブフロー ID ( %0 ) は存在しますが、指定されたプロセス ID (%1) の子ではありません。"},{"text_key":"engine-util-sbfl-not-found","source":"Subflow ID supplied ( %0 ) not found. Check for process events that changed process flow (timeouts, errors, escalations).","target":"指定されたサブフロー ID ( %0 ) が見つかりませんでした。プロセス・フローを変更したプロセス・イベント (タイムアウト、エラー、エスカレーション) をチェックしてください。"},{"text_key":"gateway-invalid-route","source":"Error at gateway %0. Supplied variable %1 contains invalid route: %2","target":"ゲートウェイ %0 でエラーが発生しました。指定された変数に無効なルートが含まれています: %2"},{"text_key":"archive-destination-null","source":"No archive or logging destination has been configured - See Configurations > Logging or Archiving","target":"アーカイブまたはロギングの宛先が構成されていません- 「構成」&gt;「ロギング」または「アーカイブ」を参照してください"},{"text_key":"var_exp_plsql_other","source":"Error setting process variable %1 in process id %0 (set %2). PL/SQL error shown in event log. ","target":"プロセスID %0 (セット%2)でプロセス変数%1を設定中にエラーが発生しました。イベント・ログにPL/SQLエラーが表示されます。"},{"text_key":"rewind-parent-not-call-act","source":"Rewind - parent subflow is not a call activity to rewind to.","target":"巻き戻し- 親サブフローは巻き戻すコール・アクティビティではありません。"},{"text_key":"gateway-no-previous-gw","source":"No previous opening parallel or inclusive gateway found for subflow %0.","target":"サブフロー%0の前のオープン・パラレル・ゲートウェイまたは包含ゲートウェイが見つかりません。"},{"text_key":"resume-invalid-status","source":"Only process instances currently in suspended status can be resumed.","target":"現在一時停止中ステータスのプロセス・インスタンスのみを再開できます。"},{"text_key":"rewind-no-matched-object","source":"Rewind - no matching opening object found to rewind to.","target":"巻き戻し- 巻き戻すための一致する開始オブジェクトが見つかりません。"},{"text_key":"rewind-unknown-gw-status","source":"Rewind - unknown gateway status on rewind to prior gateway. ","target":"巻き戻し- 以前のゲートウェイへの巻き戻し時に不明なゲートウェイステータス。"},{"text_key":"rewind-no-previous-gateway","source":"Rewind - no previous gateway found to rewind to.","target":"巻き戻し- 巻き戻す前のゲートウェイが見つかりません。"},{"text_key":"rewind-no-parent-call-act","source":"Rewind - no parent call activity subflow found to rewind to.","target":"巻き戻し- 巻き戻し先の親コール活動サブフローが見つかりません。"},{"text_key":"rewind-split-event-based-gw","source":" Rewind - cannot rewind from an event based gateway that is still running. Rewind one of the forward subflows back to the event based gateway first.","target":" 巻き戻し- まだ実行中のイベントベースのゲートウェイから巻き戻すことはできません。フォワード・サブフローの1つを最初にイベント・ベースのゲートウェイに巻き戻します。"},{"text_key":"exec_plsql_other","source":"Error executing PL/SQL. PL/SQL error shown in event log.","target":"PL/SQLの実行中にエラーが発生しました。イベント・ログにPL/SQLエラーが表示されます。"},{"text_key":"gateway-matching-object-error","source":"Error retrieving matching gateway object for subflow %0, current step %1.","target":"サブフロー%0、現在のステップ%1の一致するゲートウェイ・オブジェクトの取得中にエラーが発生しました。"},{"text_key":"exec_sql_too_many_cols","source":"Error executing SQl Query - query returns more than one column.","target":"SQl問合せの実行中にエラーが発生しました- 問合せは複数の列を返します。"},{"text_key":"rewind-no-parent-subflow","source":"Rewind - no parent subflow found.","target":"巻き戻し- 親サブフローが見つかりません。"},{"text_key":"exec_sql_too_many_rows","source":"Error setting process variable %1 in process id %0 (set %2). Query returns multiple rows.","target":"プロセスID %0 (セット%2)でプロセス変数%1を設定中にエラーが発生しました。問合せは複数の行を返します。"},{"text_key":"rewind-cant-gway-from-event","source":"Rewind - cannot rewind to previous gateway from event.","target":"巻き戻し- イベントから前のゲートウェイに巻き戻すことはできません。"},{"text_key":"logging-event-level","source":"Flows - Internal error while getting the logging level","target":"フロー- ロギング・レベルの取得中に内部エラーが発生しました"},{"text_key":"rewind-not-wait-at-gateway","source":"Rewind - subflow is not waiting at a gateway to rewind to last step.","target":"巻き戻し- サブフローはゲートウェイで最後のステップに巻き戻すのを待機していません。"},{"text_key":"exec_sql_too_many_values","source":"Error executing SQL Query - query returns more than one value.","target":"SQL問合せの実行中にエラーが発生しました- 問合せで複数の値が返されました。"},{"text_key":"timer-internal-error","source":"Timer internal error for object %0. Type: %1; Value: %2","target":"オブジェクト%0のタイマー内部エラーです。タイプ: %1; 値: %2"},{"text_key":"suspend-invalid-status","source":"Only process instances currently in running or error status can be suspended.","target":"現在実行中のプロセス・インスタンスまたはエラー・ステータスのプロセス・インスタンスのみを一時停止できます。"},{"text_key":"rewind-force-step-not-error","source":"Cannot force next step on subflow that is not in error status.","target":"エラー・ステータスではないサブフローで次のステップを強制できません。"},{"text_key":"gateway-multiple-previous-gws","source":"Multiple previous opening parallel or inclusive gateways found for subflow %0.","target":"サブフロー%0に対して複数の直前のオープン・パラレル・ゲートウェイまたは包含ゲートウェイが見つかりました。"},{"text_key":"suspended-cannot-complete-step","source":"This process instance is currently suspended. Please try again later.","target":"このプロセス・インスタンスは現在一時停止中です。後で再試行してください。"},{"text_key":"rewind-no-last-step","source":"Rewind - no last completed step found immediately before gateway to rewind to. ","target":"巻き戻し- 元に戻すゲートウェイの直前に最後に完了したステップが見つかりません。"},{"text_key":"rewind-step-reposition-error","source":"Rewind - error repositioning step.","target":"巻き戻し- ステップの再配置中にエラーが発生しました。"},{"text_key":"rewind-no-current-step","source":"Rewind - requested target step for rewind is not a valid transition from current step.","target":"巻き戻し- 要求された巻き戻しのターゲット・ステップは、現在のステップからの有効な遷移ではありません。"},{"text_key":"rewind-matching-object-error","source":"Rewind - error getting matching object for rewind from subflow log. ","target":"巻き戻し- サブフローログからの巻き戻しのための一致するオブジェクトを取得中にエラーが発生しました。"},{"text_key":"rewind-parent-not-subproc","source":"Rewind - parent subflow is not a subprocess to rewind to.","target":"巻き戻し- 親サブフローは巻き戻すサブプロセスではありません。"},{"text_key":"exec_sql_other","source":"Error executing SQL Query. SQL error shown in event log.","target":"SQL問合せの実行中にエラーが発生しました。イベント・ログにSQLエラーが表示されます。"}]}